src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrForwardFlower.cs

```diff
@@ -267,14 +270,20 @@ protected override async Task<bool> OppositeDirectionFlowAsync(
         bool headBranchExisted,
         CancellationToken cancellationToken)
     {
-        await sourceRepo.CheckoutAsync(lastFlow.RepoSha);
-
-        var patchName = _vmrInfo.TmpPath / $"{headBranch.Replace('/', '-')}.patch";
+        // When updating an existing PR, we create a work branch to make the changes on
+        IWorkBranch? workBranch = null;
         var branchName = currentFlow.GetBranchName();
+        var vmr = _localGitRepoFactory.Create(_vmrInfo.VmrPath);
+        if (headBranchExisted)
+        {
+            // Check out the last flow's commit in the PR branch to create the work branch on
+            await vmr.CheckoutAsync(lastFlows.LastForwardFlow.VmrSha);
```

Comment:
this doesn't include any manual changes, right? It's just what the service has flown